imports/plugins/core/accounts/client/templates/addressBook/add/add.js

```diff
@@ -1,126 +0,0 @@
-import { $ } from "meteor/jquery";
-import { Session } from "meteor/session";
-import { Meteor } from "meteor/meteor";
-import { HTTP } from "meteor/http";
-import { ReactiveVar } from "meteor/reactive-var";
-import { AutoForm } from "meteor/aldeed:autoform";
-import { Template } from "meteor/templating";
-import { Reaction, i18next } from "/client/api";
-import * as Collections from "/lib/collections";
-
-Template.addressBookAdd.onCreated(function () {
-  this.currentCountry = new ReactiveVar(null);
-  // hit Reaction's GeoIP server and try to determine the user's country
-  HTTP.get("https://geo.getreaction.io/json/", (err, res) => {
```

Comment:
This HTTP call was hitting our GeoIP server to get the country the user is currently in and automatically set their country in the select field.  It looks like this got ditched completely.  Do we have a suitable alternative to maintain that UX?

Reply:
@jshimko No there wasn't. Now I have added it.